presentation/src/main/java/com/sopt/presentation/calendar/component/bottomsheet/ScheduleItem.kt

```diff
@@ -0,0 +1,49 @@
+package com.sopt.presentation.calendar.component.bottomsheet
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.sopt.core.designsystem.theme.NoostakTheme
+import com.sopt.core.extension.noRippleClickable
+import com.sopt.domain.entity.ScheduleDetailEntity
+
+@Composable
+fun ScheduleItem(
+    data: ScheduleDetailEntity,
+    onItemClick: (ScheduleDetailEntity) -> Unit = {}
+) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
```

Comment:
P1: 글자 길어져도 세로 막대기는 위에 고정이라서 Top으로 수정 ㄱㄱ
```suggestion
        verticalAlignment = Alignment.Top,
```

Reply:
다르게 수정햇슈